app/models/container_node.rb

```diff
@@ -78,4 +79,19 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  supports :common_logging do
+    unless ext_management_system.respond_to?(:common_logging_route_name)
+      unsupported_reason_add(:common_logging, _('This provider type does not support common_logging'))
+    end
+  end
+
+  def common_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def common_logging_path
+    hostname = name.split('.')[0]
```

Comment:
@enoodle @portante do we know officially what this field `hostname` should contain?
Isn't there another field to filter by where the FQDN is used?

I know chances are low... but aren't you worried of collisions of hostnames if you're not using FQDN? (And also with FQDN there could be collisions, but there it's more a deployment issue).